ReactNativeClient/lib/services/rest/Api.js

```diff
@@ -451,6 +451,24 @@ class Api {
 			return note;
 		}
 
+		if (request.method === 'PUT') {
+			const note = await Note.load(id);
+
+			if (!note) throw new ErrorNotFound();
+
+			let updatedNote = await this.defaultAction_(BaseModel.TYPE_NOTE, request, id, link);
+
+			const requestNote = JSON.parse(request.body);
+			if (requestNote.tags === '') {
+				await Tag.setNoteTagsByTitles(id, '');
+			} else if (requestNote.tags) {
+				const tagTitles = requestNote.tags.split(',');
+				await Tag.setNoteTagsByTitles(id, tagTitles);
+			}
```

Comment:
Can this be reduced to a single if statement?  I don't think you need two branches.

Reply:
Yes, I just realized two branches weren't necessary. I changed the code to use just a single `if`.